src/models/admin.ts

```diff
@@ -0,0 +1,20 @@
+import { DataTypes } from 'sequelize';
+import { AdminAttributes } from '../types/index';
```

Comment:
no need to write the full file path

import { AdminAttributes } from '../types/index';
```suggestion
import { AdminAttributes } from '../types';
```